### PR TITLE
Adding iOS XCFramework support

### PR DESCRIPTION
**Pull request by spoorthipujariadobe**
## Description

Adding XCFramework support for iOS by removing bundled iOS libraries. ACPCore XCFrameworks need to be downloaded and added to Xcode project. The steps are included in the installation instructions.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -304,28 +314,30 @@ To build demo app for specific platform follow the below instructions.
...
-## Additional Cordova Plugins
-
-Below is a list of additional Unity plugins from the AEP SDK suite:
+2. Select `iOS` from the __Platform__ window
+3. If `iOS` is not the active platform, hit the button that says __Switch Platform__ (it will only be available if you actually need to switch active platforms)
+4. Press the __Build And Run__ button
+5. You will be asked to provide a location to save the build. Make a new directory at *unity-acpcore/ACPCore/Builds* (this folder is in the .gitignore file)
+6. Name build whatever you want and press __Save__
+7. `Unity` will create and open an `Xcode` project
+8. From the Xcode project run the app on a simulator.
```
Comment (shtomar-adb) [inline README.md:333]: We can move step 8 at the end i.e after the steps for adding XCFramework to the Xcode project.

File: README.md [hunk 2]
```diff
@@ -304,28 +314,30 @@ To build demo app for specific platform follow the below instructions.
...
+9. [Download](https://github.com/Adobe-Marketing-Cloud/acp-sdks/tree/master/iOS/ACPCore) `ACPCore.xcframework`, `ACPIdentity.xcframework`, `ACPLifecycle.xcframework` and `ACPSignal.xcframework`.
+10. Select the UnityFramework target -> Go to Build Phases tab -> Add the XCFrameworks downloaded in Step 9 to `Link Binary with Libraries`.
```
Comment (shtomar-adb) [inline README.md:335]: nit: We have already added steps to add xcframework to the Xcode project. We can replace these 3 steps with one step  "add xcframework" which link to the steps above. i.e [Add XCFramework](#setup)

**Review by shtomar-adb — APPROVED** [review]
Looks good. I have left 2 comments. Feel free to merge after addressing them.

File: README.md [hunk 3]
```diff
@@ -304,28 +314,28 @@ To build demo app for specific platform follow the below instructions.
...
+7. `Unity` will create and open an `Xcode` project
+8. [Add XCFrameworks to the Xcode project](#ios-installation).
+9. From the Xcode project run the app on a simulator.
+
+## Additional Unity Plugins
+
+Below is a list of additional Unity plugins from the ACP SDK suite:
 
 | Extension | GitHub | Unity Package |
 |-----------|--------|-----|
```
Comment (emdobrin) [inline README.md:341]: for consistency, can we rename the Extension values to: ACPAnalytics, AEPAssurance and also update the versions

**Review by emdobrin — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -23,10 +23,13 @@ Plugins for a Unity project use the following folder structure:
 
 `{Project}/Assets/Plugins/{Platform}`
 
+## Building the ACPCore Unity Package
+1. Clone the current repository.
+2. In the root directory, run the following command to build the ACPCore Unity package in bin -> build_temp -> ACPCore-1.0.0-Unity -> ACPCore folder:
```
Comment (emdobrin) [inline README.md:28]: 2. In the root directory, run the following command to build the ACPCore Unity package:
command
3. In the Unity project, import `ACPCore.unitypackage` via Assets-Import Package from `bin/build_temp/ACPCore-1.0.0-Unity`.

File: README.md [hunk 5]
```diff
@@ -23,11 +23,24 @@ Plugins for a Unity project use the following folder structure:
 
 `{Project}/Assets/Plugins/{Platform}`
 
+## Building the ACPCore Unity Package
```
Comment (emdobrin) [inline README.md:26]: let's move this with \#### under the Installation instructions.

**Pull request merged**